src/components/ComputeModal.js

```diff
@@ -499,6 +502,11 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     return existingPersistentDisk && !canUpdatePersistentDisk()
   }
 
+  const isToolRStudio = () => {
+    const { runtime: existingRuntime } = getExistingEnvironmentConfig()
+    return existingRuntime.tool === 'RStudio'
```

Comment:
I think `getExistingEnvironmentConfig` is providing adequate information, so this is clearer when shortened to:
```suggestion
    return getExistingEnvironmentConfig().runtime.tool === 'RStudio'
```